core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -235,7 +235,8 @@ class SocketServerTest extends JUnitSuite {
     val trustStoreFile = File.createTempFile("truststore", ".jks")
     val overrideProps = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, enableSsl = true,
       trustStoreFile = Some(trustStoreFile))
-    overrideProps.put("listeners", "SSL://localhost:0")
+    overrideProps.put(KafkaConfig.ListenersProp, "SSL://localhost:0")
+    overrideProps.put(KafkaConfig.InterBrokerSecurityProtocolProp, SecurityProtocol.SSL)
```

Comment:
@ijuma createBrokerConfig is used for the all the tests, this test is overriding to limit to SSL. Happy to rework things if you think I should. I tried to keep the test change minimal.